packages/cip2/src/selectionConstraints.ts

```diff
@@ -32,10 +38,7 @@ export const computeMinimumCost =
   };
 
 export const computeMinimumCoinQuantity =
-  (
-    csl: CardanoSerializationLib,
-    { coinsPerUtxoWord }: ProtocolParametersRequiredByWallet
-  ): ComputeMinimumCoinQuantity =>
+  (csl: CardanoSerializationLib, coinsPerUtxoWord: number): ComputeMinimumCoinQuantity =>
```

Comment:
```suggestion
  (csl: CardanoSerializationLib, coinsPerUtxoWord: ProtocolParametersRequiredByWallet['coinsPerUtxoWord']): ComputeMinimumCoinQuantity =>
```
It's more verbose but is less brittle and improves documentation

Reply:
So it would be better to check for null and throw in each constraint instead of in `defaultSelectionConstraints`?

Reply:
My suggestion here is just to use an alias for the function parameter type, so we know arguments being passed in are expected to be part of the `ProtocolParametersRequiredByWallet` type